tools/sts-job-manager/lib/table_util.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The file contains a list of table-related operations.
+"""
+
+from typing import List
+
+from google.cloud import bigquery
+
+from lib.options import BigQueryOptions
+from lib.services import Services
+
+
+def create_table(client: Services.bigquery, options: BigQueryOptions,
+                 table_name: str, schema: List[bigquery.SchemaField]):
+    """
+    Creates tables. Does not throw if it exists.
+    """
+
+    print("Creating `{}` table if it does not exists...".format(table_name))
```

Comment:
Please make sure you are using a Logger library instead of print. If this is run inside a container i'm not sure you will proper structure logging. I'm just trying to be mindful to the use-case where people will want to run this on GCE or GKE. There is an example here https://github.com/GoogleCloudPlatform/professional-services/blob/master/tools/slo-generator/slo_generator/compute.py

Reply:
Good point - I'll switch to `logging`

Reply:
Great, let me know when you change to logging and i will review it again. Thanks

Reply:
Updated - since enough code has changed I'll rerun a few e2e tests to ensure everything still works as expected. Then I'll follow-up with confirmation

Reply:
Ok, we look gtg